content/source/docs/extend/best-practices/naming.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: "extend"
+page_title: "Extending Terraform: Best Practices"
+sidebar_current: "docs-extend-best-practices-naming"
```

Comment:
We just need a sidebar entry, you can add it here:
https://github.com/hashicorp/terraform-website/blob/master/content/source/layouts/extend.erb#L45

Also super minor nitpick, it looks like so far the entries and paragraphs on the index page are in alphabetical order. I don't mind having naming be first though, I actually prefer the content flows from "beginner stuff --> advanced stuff", but I will leave it up to you I really don't find that matters.

Reply:
Yeah I thought naming was kind of a foundational one, so I thought first made sense.